specification/marketplacecatalog/data-plane/Search/preview/2022-09-25-preview/search.json

```diff
@@ -0,0 +1,2229 @@
+            "name": "searchQuery",
+            "in": "query",
+            "description": "The search text, if no value selected, this filter is ignored. Such as searchQuery=red hat"
+          },
+          {
+            "type": "string",
+            "name": "publisherDisplayName",
+            "in": "query",
+            "description": "The search publisher name, if no value selected, this filter is ignored. Such as publisherDisplayName=red hat"
+          },
+          {
+            "type": "string",
+            "name": "azureBenefit",
+            "in": "query",
+            "description": "Define the search for only azure benefit eligible offers, if no value selected, this filter is ignored. Default: null. \n- `Eligible`: Eligible.\n- `NotEligible`: Not Eligible.",
+            "x-schema": {
+              "$ref": "#/definitions/AzureBenefit"
+            },
+            "enum": [
+              "Eligible",
+              "NotEligible"
+            ],
+            "x-ms-enum": {
+              "name": "AzureBenefit",
+              "modelAsString": true
+            }
+          },
+          {
+            "type": "array",
+            "name": "publisherTypes",
+            "in": "query",
+            "description": "Define the search publisher types, if no value selected, this filter is ignored. Possible values: Microsoft, ThirdParty. Default: Microsoft. \n- `Microsoft`: Microsoft.\n- `ThirdParty`: Third Party.",
+            "collectionFormat": "csv",
+            "items": {
+              "type": "string",
+              "enum": [
+                "Microsoft",
+                "ThirdParty"
+              ],
+              "x-ms-enum": {
+                "name": "PublisherType",
+                "modelAsString": true
+              }
+            }
+          },
+          {
+            "type": "array",
+            "name": "badges",
+            "in": "query",
+            "description": "Products that contains any of the given badges. If no value selected, this filter is ignored. The following product badges are available: \n- `PreferredSolution`: Preferred Solution.\n- `PowerBICertified`: power BI Certified.\n- `AdditionalPurchaseRequirement`: Additional Purchase Requirement.",
+            "collectionFormat": "csv",
+            "items": {
+              "type": "string",
+              "enum": [
+                "PreferredSolution",
+                "PowerBICertified",
+                "AdditionalPurchaseRequirement"
+              ],
+              "x-ms-enum": {
+                "name": "Badge",
+                "modelAsString": true
+              }
+            }
+          },
+          {
+            "type": "string",
+            "name": "industryCloud",
+            "in": "query",
+            "description": "Define the search for IndustryCloud offers. If no value selected, this filter is ignored. Default: NotApplicable. \n- `NotApplicable`: Not Applicable.\n- `True`: True.\n- `False`: False.",
```

Comment:
I am unable to parse this description?

Reply:
@johanste  what do you mean? seems fine in swagger editor and in json validator..it passed all validation tools as well